mlir/lib/Dialect/Tosa/Transforms/TosaValidation.cpp

```diff
@@ -1217,9 +1217,10 @@ bool TosaValidation::isValidElementType(Type type, const bool allowUnsigned) {
         return true;
       }
     }
-  } else if (mlir::isa<tosa::shapeType>(type)) {
+  } else if (mlir::isa<tosa::shapeType>(type))
```

Comment:
NIT: this type check for `tosa::shapeType` is inconsistent with the rest of the function, which uses isa<...>(type)